src/commons/crypto/signing/signers/pkcs11/internal.rs

```diff
@@ -173,7 +173,11 @@ impl Pkcs11Signer {
         // AWS CloudHSM library, presumably they both need initlaizing within the same instance of the Krill
         // "application".
 
-        let server = Arc::new(StatefulProbe::new(Arc::new(conf.try_into()?), Duration::from_secs(30)));
+        let server = Arc::new(StatefulProbe::new(
+            name.to_string(),
+            Arc::new(conf.try_into()?),
+            Duration::from_secs(30),
```

Comment:
similar to kmip I wonder if the probe interval should be part of the config (with a sensible default of course)

Reply:
I linked to this comment from #726.